wikichange/src/content/content.js

```diff
@@ -1,78 +1,97 @@
-import { WIKI_CREATION_DATE, AggregateType } from "./enums.js";
-import { getPageViews, getPageCreationDate } from "./timeSeriesService.js";
+import { getPageCreationDate } from "./timeSeriesService.js";
+import injectGraphToPage from "./graph.js";
 
 const insertAfter = (newNode, existingNode) => {
     existingNode.parentNode.insertBefore(newNode, existingNode.nextSibling);
-}
+};
+
+/* Get the title of a Wikipedia page by inspecting the html */
+const title = (() => {
+    let titleSpan = document.getElementsByClassName("mw-page-title-main");
+    let title = titleSpan[0].innerHTML;
+    return title;
+})();
 
 /* Creates the div for the graph overlay. TODO: create the graph and render it here */
-const renderGraphOverlay = () => {
-    let floatContainer = document.createElement('div');
-    floatContainer.style.cssText = 'display: flex;';
-    floatContainer.setAttribute('id', 'floatContainer');
-
-    let graphContainer = document.createElement('div');
-    graphContainer.setAttribute('id', 'graphOverlay');
-
-    let canvas = document.createElement('canvas');
-    canvas.style.maxHeight = '200px';
-    canvas.id = 'viewsEditsChart';
-    graphContainer.style.cssText = 'width:75%;height:20%;';
+const renderGraphOverlay = async () => {
+    let floatContainer = document.createElement("div");
+    floatContainer.style.cssText = "display: flex;";
+    floatContainer.setAttribute("id", "floatContainer");
+
+    let graphContainer = document.createElement("div");
+    graphContainer.setAttribute("id", "graphOverlay");
+
+    let canvas = document.createElement("canvas");
+    canvas.style.maxHeight = "200px";
+    canvas.id = "viewsEditsChart";
+    graphContainer.style.cssText = "width:75%;height:20%;";
     graphContainer.appendChild(canvas);
 
     floatContainer.appendChild(graphContainer);
 
-    let p = document.createElement('p');
+    let p = document.createElement("p");
     graphContainer.appendChild(p);
 
-    let siteSub = document.getElementById('siteSub');
+    let siteSub = document.getElementById("siteSub");
     insertAfter(floatContainer, siteSub);
-}
+    const creationDate = await getPageCreationDate();
+    injectGraphToPage(title, creationDate, new Date(Date.now()));
```

Comment:
Added these lines. can change the start date to something that is not `creationDate` later.